custom_components/reflex_clerk_api/clerk_provider.py

```
@@ -313,26 +328,50 @@ def add_custom_code(self) -> list[str]:
+        addEvents([ReflexEvent("%s.clear_clerk_session")])
       }
-  }, [isSignedIn])
+  }, [isLoaded, isSignedIn, addEvents, getToken])
 
   return (
       <>{children}</>
   )
 }
 """
-            % (clerk_state_name, clerk_state_name)
+            % (clerk_state_name, clerk_state_name, clerk_state_name, clerk_state_name)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)